components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -84,6 +86,12 @@ exec $HAB_ROOT_PATH/bin/hab pkg exec core/hab-plan-build hab-plan-build \$*
 EOF
   $bb chmod $v 755 $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/build
 
+  $bb cat <<TAIL_SUP > $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/tail_sup
```

Comment:
What about just simply calling it `log` or `logs` like how we have the `build` command built in?